interp/interp_test.go

```diff
@@ -25,6 +25,15 @@ import (
 	"mvdan.cc/sh/v3/syntax"
 )
 
+// Some program which should be in $PATH.  Needs to run before runTests is
+// initialized, below.
```

Comment:
explain why - I assume it's because we want to use it in "expected output" string literals